cfme/tests/configure/test_docs.py

```diff
@@ -67,6 +67,20 @@ def docs_info():
             'User',
             'Settings and Operations'
         ]
+    elif version.appliance_is_downstream():
+        return [
+            'Monitoring Alerts Reporting',
+            'General Configuration',
+            'Virtual Machines Hosts',
+            'Methods For Automation',
+            'Infrastructure Inventory',
+            'Providers',
+            'Scripting Actions',
+            'Defining Policies Profiles'
+        ]
+    else:
+        # Upstream version has no docs
+        return []
```

Comment:
I think About page is accessed in another test as far as I recall, I also agree we should probably skip on upstream using uncollectif.